docker-mender-dist-packages

```diff
@@ -78,3 +78,16 @@ docker run --rm \
         ${MENDER_CONFIGURE_VERSION:-master} \
         amd64 \
         ${CI_PIPELINE_ID:-LOCAL}
+
+# mender-monitor
+if [ -n "${MENDER_TEST_BOT_ACCESS_TOKEN}" ]; then
```

Comment:
I have a suggestion for this but be aware that it might break your bash magic for `REPO_URL_CLEANED`.

I generally don't like having hardcoded the user (mender-test-bot) in this script. Basically we are encouraging a developer to export bot credentials to his/her console in order to debug this package. Instead both the user and the token should be bash variables so that we encourage a developer to export his/her own credentials (creating a temporary token) for debugging.

That being said, this could be something like:
```suggestion
if [ -n "${MENDER_PRIVATE_REPO_ACCESS_USER}" -a -n "${MENDER_PRIVATE_REPO_ACCESS_TOKEN}" ]; then
```

Having in the gitlab-ci.yml file `MENDER_PRIVATE_REPO_ACCESS_USER` set to `mender-test-bot`, for example

Reply:
Yes, good idea!